### PR TITLE
Update fingerterm

### LINKED ISSUE #242: Update fingerterm to newest source?
Hi, I wasn't sure how else to do this since I couldn't find a similar request in the issues.

But  dixonary's fingerterm-reMarkable repo has had a lot of updates (toltec timestamp says 2017), namely being able to turn off the large feedback when using the keyboard by adding

```
showVisualKeyPressFeedback=false
keyPressFeedback=false
```

to `[ui]` in `/home/root/.config/FingerTerm/settings.ini`

Should I send a PR into testing with updated SHA's? Or is it best if we send in issues for the package devs/toltec maintainers to handle.

### PR DESCRIPTION
**Pull request by matteodelabre**
(Fixes #242)

This adds commit <https://github.com/dixonary/fingerterm-reMarkable/commit/02c17b5b485743c698e005ca89366c32b66aa044> which provides a new setting called 'showVisualKeyPressFeedback' for disabling the flash that happens after each tap on a key of the terminal keyboard.

### CODE REVIEW

File: package/fingerterm/package [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
 url=https://github.com/dixonary/fingerterm-reMarkable
-pkgver=1.3.5-11
-timestamp=2017-12-08T15:40Z
+pkgver=1.3.5-12
+timestamp=2020-10-27T12:02Z
 section="admin"
 maintainer="Mattéo Delabre <spam@delab.re>"
 license=GPL-2.0-or-later
 
 image=qt:v1.1
```
Comment (LinusCDE) [inline package/fingerterm/package]:
```suggestion
image=qt:v1.2.2
```

Might upgrade this as well while at it. (Tested with both versions (and each on both models), so I'll can approve it no matter what version gets used.)

**Review by LinusCDE — DISMISSED** [review]
Works on both rM1 and rM2. Tested with the new visual option. I wanted something like it for a while! :+1: 

I also re-tested with using qt:v1.2.2. Might throw this upgrade in there as well.

File: package/fingerterm/package (hunk 1 above)
Comment (matteodelabre) [reply package/fingerterm/package]: Good point. Thanks for testing!

**Comment by matteodelabre** [comment]
Thanks for your review @LinusCDE. You’ll need to re-approve since I committed your suggestion to the branch.

**Review by LinusCDE — APPROVED** [review]
Sure. Wasn't sure if this would dismiss it since I approved with that suggestion.

**Pull request merged**